bin/logstash.lib.sh

```diff
@@ -111,6 +111,7 @@ setup_java() {
       else
         echo "Invalid JAVA_HOME, doesn't contain bin/java executable."
       fi
+      echo "DEPRECATION: Using JAVA_HOME, please configure LS_JAVA_HOME instead."
```

Comment:
Now that I read this, I think we need to improve it.

DEPRECATION: The use of JAVA_HOME is now deprecated and will be soon removed, please configure LS_JAVA_HOME instead.

Now, we could also say in which release we would deprecate, e.g.:

DEPRECATION: The use of JAVA_HOME is now deprecated and will be removed starting from 8.0. Please configure LS_JAVA_HOME instead

Reply:
Sounds better